rosidl_typesupport_introspection_c/include/rosidl_typesupport_introspection_c/message_introspection.h

```diff
@@ -53,17 +75,28 @@ typedef struct rosidl_typesupport_introspection_c__MessageMember_s
   ///
   /// Available for array and sequence members.
   void (* assign_function)(void *, size_t index, const void *);
+  /// If is_array_ is true, a pointer to a function that resizes the array.
+  /// Only used for nested interface types.
```

Comment:
Same comment about being used for more than nested interface types

Reply:
c740f29